applications/tari_base_node/src/builder.rs

```diff
@@ -497,7 +497,7 @@ where
 
     //---------------------------------- Base Node --------------------------------------------//
 
-    let (publisher, base_node_subscriptions) = pubsub_connector(handle.clone(), 1500);
+    let (publisher, base_node_subscriptions) = pubsub_connector(handle.clone(), 1_500);
```

Comment:
Maybe this should be in the config file, with this default if not specified

Reply:
See comment below